ui/apps/platform/src/constants/violationStates.js

```diff
@@ -1,8 +1,13 @@
-const VIOLATION_STATES = Object.freeze({
+export const VIOLATION_STATES = Object.freeze({
     ACTIVE: 'ACTIVE',
     ATTEMPTED: 'ATTEMPTED',
     RESOLVED: 'RESOLVED',
     SNOOZED: 'SNOOZED',
 });
 
-export default VIOLATION_STATES;
+export const VIOLATION_STATE_LABELS = Object.freeze({
```

Comment:
👀  TIL that `Object.freeze()` types the object similarly to `as const` when used in JavaScript files.